src/containers/Storage/Storage.tsx

```diff
@@ -155,19 +174,23 @@ export const Storage = ({additionalNodesInfo, tenant, nodeId}: StorageProps) =>
                     <StorageGroups
```

Comment:
Lets make an issue to refactor Storage component - now its a little mess with different storage types, as for me it should be splitted.

Reply:
https://github.com/ydb-platform/ydb-embedded-ui/issues/511